pkg/zpdb/eviction_controller.go

```diff
@@ -28,6 +28,20 @@ const (
 	logAllowMesg = "pod eviction allowed"
 )
 
+// MaxUnavailableZeroOverride provides an alternate maxUnavailable value to be used when the current maxUnavailable configuration is set to zero
```

Comment:
Idiomatic ending period:
```suggestion
// MaxUnavailableZeroOverride provides an alternate maxUnavailable value to be used when the current maxUnavailable configuration is set to zero.
```